lib/marginalia/comment.rb

```diff
@@ -18,11 +20,11 @@ def self.update_adapter!(adapter)
     end
 
     def self.construct_comment
-      ret = ''
+      ret = String.new
       self.components.each do |c|
         component_value = self.send(c)
         if component_value.present?
-          ret << "#{c.to_s}:#{component_value.to_s},"
+          ret << "#{c}:#{component_value},"
```

Comment:
I removed the `.to_s` calls since string interpolation already calls `to_s` on its values